src/mpi/info/infoutil.c

```diff
@@ -65,3 +65,12 @@ int MPIR_Info_alloc(MPIR_Info ** info_p_p)
   fn_fail:
     return mpi_errno;
 }
+
+/* Set up MPI_INFO_ENV.  This may be called before initialization and after
+ * finalization by MPI_Info_create_env().  This routine must be thread-safe. */
+void MPIR_Info_setup_env(MPIR_Info * info_ptr)
```

Comment:
Thanks! `MPII_init_local_proc_attrs()`, which is the other caller of `MPIR_Info_setup_env()`, this does not take `argc` and `argc` at this point. To be honest, I don't know what parameters will be needed to set up this info object; for example, Fortran passes `NULL` for both so perhaps such arguments will not be necessary to initialize this info object. Since now the created info object is empty, I believe we can keep it as it is.